### ISSUE TITLE
[Bug] no image in usb macos (m1)

### ISSUE THREAD

**Issue by Reaster0** [exelix11/SysDVR#154]
[To submit a bug report fill all the parts of the template, then remove all the text within square brackets "[ ]" ]
[Issues that don't follow the template will be closed]

**Describe the bug**
[the window is only black, sound is okay nonetheless, the program print the error :
 "libusb: error [submit_bulk_transfer] bulk transfer failed (dir = In): pipe is stalled (code = 0xe000404f)" ]

**To Reproduce**
[launch the app using dotnet on macos with or without the usb arg because it dont change anything]

**Expected behavior**
[the window launch with sound but without any image and print the error message again and again]

**Setup information**
 - PC Os: [macos 12 (m1)]
 - Console firmware [13.0.0]
 - Custom firmware name and version [AMS 1.1.1]
 - SysDVR version [5.1]

**Additional context**
[on a windows pc i dont have any issues and i dont have the possibility to try the other way to connect than usb
<img width="809" alt="Capture d’écran 2021-09-26 à 20 08 39" src="https://user-images.githubusercontent.com/73885131/134823663-0d4034ad-c2c9-4ac2-905a-1a7c0215742f.png">
]

**Comment by corando98** [comment]
Having this same issue on an M1 Pro. Did you find any solution

**Comment by Reaster0** [comment]
nope, still haven't found any solutions

**Comment by definev** [comment]
Me getting this error too, i suppose the libusb-1.0.0.dylib is rossetta and not work on M1 or something?

**Comment by exelix11** [comment]
Just noticed, duplicate of https://github.com/exelix11/SysDVR/issues/166
Will close this as the other one has a better title. 

TLDR currently libusbdotnet doesn't bundle native libs for arm m1 and rosetta doesn't seem to handle well x86 libusb. Guess for the time being you'll only be able to use network streaming.
